pkg/constants/args.go

```diff
@@ -18,6 +18,7 @@ const (
 	ArgDashboard               = "dashboard"
 	ArgDashboardListen         = "dashboard-listen"
 	ArgDashboardPort           = "dashboard-port"
+	ArgDefaultConfig           = "default-config"
```

Comment:
rename it to `skip-config`
```suggestion
	ArgSkipConfig           = "skip-config"
```